student.library/src/main/java/tds/student/sql/data/ItemResponseScorable.java

```diff
@@ -32,22 +32,28 @@
   private String _value;
   private UUID   _scoreMark;
   private String _filePath;
+  private String clientName;
 
   public ItemResponseScorable () {
   }
 
   public ItemResponseScorable (String testKey, String testID, String language, int position, int sequence, long bankKey, long itemKey, String segmentID, String value, UUID scoreMark, String itemFilePath) {
-    _testKey = testKey;
-    _testID = testID;
-    _language = language;
-    _position = position;
-    _sequence = sequence;
-    _bankKey = bankKey;
-    _itemKey = itemKey;
-    _segmentID = segmentID;
-    _value = value;
-    _scoreMark = scoreMark;
-    _filePath = itemFilePath;
+    this(testKey, testID, language, position, sequence, bankKey, itemKey, segmentID, value, scoreMark, itemFilePath, "");
+  }
+
+  public ItemResponseScorable(String _testKey, String _testID, String _language, int _position, int _sequence, long _bankKey, long _itemKey, String _segmentID, String _value, UUID _scoreMark, String _filePath, String clientName) {
```

Comment:
Not a big deal but the other constructor and the clientName don't have underscores to start them.  Probably good to be consistent